taskcluster/ci/config.yml

```diff
@@ -117,6 +117,12 @@ taskgraph:
             default-repository: https://github.com/mozilla-extensions/news-disinformation-study
             default-ref: master
             type: git
+        proxymonitor:
+            name: "Proxy Monitor extension"
+            project-regex: proxymonitor$
+            default-repository: https://github.com/mozilla-extensions/proxy-monitor
```

Comment:
This looks like a private repo, so we'll need `git@` URL formatting or the task will hang trying to clone.